### DISCUSSION TITLE
Favorite Hobbies

### DISCUSSION THREAD

**Discussion by Allopezgo** [unaizorrilla-githubschool/AccionaNonDevelopers#21] [General]
# _Hobbies_ **Favoritos**

**Vamos a destacar los hobbies favoritos**

El objetivo es revisar las preferencias del equipo:

> Lista

- Actividades Deportivas
- Actividades Culturales
- Actividades Formativas
- Viajes

**Comment by Allopezgo** [comment]
# Comentario de  @Allopezgo

**Comment by GVilG** [comment]
**1.Actividades Deportivas**
**2.Actividades Culturales**
**3.Actividades Formativas**

**Reply by jsagues89** [reply]
👍
